persimmon/website/views.py

```diff
@@ -93,16 +99,35 @@ def security_check(request, myusername, level):
 
     return HttpResponse("success this user match the required permissions")
 
-#index for website/ to check if url views are working
+
+# index for website/ to check if url views are working
 def index(request):
     return HttpResponse("Hello world")
 
+
+@api_function
+def create_user_account(request, username: str, first_name: str,
+                        last_name: str, password: str, email: str,
+                        phone: str, address: str, employee_level: EmployeeLevel):
```

Comment:
The public-facing endpoint should not allow users to register themselves as anything other than customers :)